utils/src/ooinstall/cli_installer.py

```diff
@@ -133,6 +133,7 @@ def collect_hosts(oo_cfg, existing_env=False, masters_set=False, print_summary=T
         if not masters_set:
             if click.confirm('Will this host be an OpenShift Master?'):
                 host_props['roles'].append('master')
+                host_props['roles'].append('etcd')
```

Comment:
It should be, just not in the interactive mode (I don't think it ever was there)